CONFIGURATION.md

```diff
@@ -0,0 +1,174 @@
+# Configuration Guide
+
+This guide explains how to create and customize configurations for the annotation tool. Configurations define the standardized variables and vocabularies available to users during the annotation process. All configurations are centrally managed and distributed through the [Neurobagel communities repository](https://github.com/neurobagel/communities).
+
+## Quick Setup
+
+To create a new configuration:
+
+1. Create a new directory and name it after the name of your configuration (e.g., `Enigma PD`)
```

Comment:
since we'll need to use these config directories in different tools/potentially in automated workflows, should we encourage the use of sanitized directory/filenames without any whitespace or special characters like `Enigma_PD` instead of `Enigma PD`?

Reply:
I can't see a reason why not. I'll update the example here but perhaps we should mention that on the official docs.